modules/ROOT/nav.adoc

```diff
@@ -1,6 +1,9 @@
 * Introduction
 ** xref::teradata-vantage-engine-architecture-and-concepts.adoc[Teradata Vantage Engine Architecture and Concepts]
 
+* ClearScape Analytics^TM^
+** xref::csae.adoc[]
```

Comment:
We might want to consider whether this should be a category on getting access to vantage. 
Also the name of the file most likely we might want to keep it uniform with other quickstarts something like, 
getting-started-with-clearscape-analytics-experience.adoc

Reply:
I agree with Daniel. Let's move it to Getting Access to Vantage section. 

There is a discussionn going on to reorgnize the content by product. Once this happens, CSAE will become its own product edition.